sanic/app.py

```diff
@@ -543,6 +543,9 @@ def run(self, host=None, port=None, debug=False, ssl=None,
         :param protocol: Subclass of asyncio protocol class
         :return: Nothing
         """
+        if sock is None:
+            host, port = host or "127.0.0.1", port or 8000
```

Comment:
this should be unneeded with https://github.com/channelcat/sanic/pull/742/files

Can you rebase this change against the HEAD?